### ISSUE TITLE
 prediction output image is totally white

### ISSUE THREAD

**Issue by whrenstone** [ugent-korea/pytorch-unet-segmentation#5]
@PyeongKim  Appreciate for your contribution. I have retrained your UNet model for line detection. My input images are gray images in png format, and the foregrounds are labelled as 0 and backgrounds are 255. However, after 2000 epoch's training, the valid output  image is totally white(all 255),  and it's also weird that Train loss reaches 0.07544739878712556 and Train accuracy reaches 0.9856020456409712  in the very beginning of training epoch. I am new to UNet and i have tried many methods on google with no big improvement, so i wish you can give me some suggestions for debugging this problem. Thanking you !  :）

**Comment by PyeongKim** [comment]
I will have a look! Sorry for late reply

**Comment by Alexander0211** [comment]
I meet the same error as you, Have you sloved it? Could you give me some advice to slove this problem? Thanks @whrenstone @PyeongKim

**Comment by Alexander0211** [comment]
I just sloved this problem. Reduce the learning rate, maybe 0.0001 is OK.
